cmd/ore/equinixmetal/create-device.go

```diff
@@ -29,30 +29,30 @@ import (
 var (
 	cmdCreateDevice = &cobra.Command{
 		Use:   "create-device [options]",
-		Short: "Create Packet device",
-		Long:  `Create a Packet device.`,
+		Short: "Create EquinixMetal device",
+		Long:  `Create an EquinixMetal device.`,
 		RunE:  runCreateDevice,
 	}
 	hostname     string
 	userDataPath string
 )
 
 func init() {
-	Packet.AddCommand(cmdCreateDevice)
+	EquinixMetal.AddCommand(cmdCreateDevice)
 	cmdCreateDevice.Flags().StringVar(&options.Facility, "facility", "sjc1", "facility code")
 	cmdCreateDevice.Flags().StringVar(&options.Plan, "plan", "", "plan slug (default board-dependent, e.g. \"baremetal_0\")")
 	cmdCreateDevice.Flags().StringVar(&options.Board, "board", "amd64-usr", "Container Linux board")
 	cmdCreateDevice.Flags().StringVar(&options.InstallerImageBaseURL, "installer-image-base-url", "", "installer image base URL, non-https (default board-dependent, e.g. \"http://stable.release.flatcar-linux.net/amd64-usr/current\")")
-	cmdCreateDevice.Flags().StringVar(&options.InstallerImageKernelURL, "installer-image-kernel-url", "", "Packet installer image kernel URL, (default installer-image-base-url/flatcar_production_pxe.vmlinuz)")
-	cmdCreateDevice.Flags().StringVar(&options.InstallerImageCpioURL, "installer-image-cpio-url", "", "Packet installer image cpio URL, (default installer-image-base-url/flatcar_production_pxe_image.cpio.gz)")
-	cmdCreateDevice.Flags().StringVar(&options.ImageURL, "image-url", "", "image base URL (default board-dependent, e.g. \"https://alpha.release.flatcar-linux.net/amd64-usr/current/flatcar_production_packet_image.bin.bz2\")")
+	cmdCreateDevice.Flags().StringVar(&options.InstallerImageKernelURL, "installer-image-kernel-url", "", "EquinixMetal installer image kernel URL, (default installer-image-base-url/flatcar_production_pxe.vmlinuz)")
+	cmdCreateDevice.Flags().StringVar(&options.InstallerImageCpioURL, "installer-image-cpio-url", "", "EquinixMetal installer image cpio URL, (default installer-image-base-url/flatcar_production_pxe_image.cpio.gz)")
+	cmdCreateDevice.Flags().StringVar(&options.ImageURL, "image-url", "", "image base URL (default board-dependent, e.g. \"https://alpha.release.flatcar-linux.net/amd64-usr/current/flatcar_production_equinixmetal_image.bin.bz2\")")
```

Comment:
Hm, this link is 404 as we still produce "packet" images, not "equinixmetal" ones.

Reply:
Good catch, I tried to preserve the URL links but I missed this one. It's just an example value but let's keep things correct. :)